### PR TITLE
Fix awacs

### PR DESCRIPTION
**Pull request by cjbehm**
The main part of this fix is to use the Unit controller instead of Group controller, but I also fixed a typo and a bug for the E-2D aircraft.

My test mission has working A-50 and E-2D detection now.

### CODE REVIEW

File: GCICAP.lua [hunk 1]
```diff
@@ -968,7 +968,12 @@ do
 
             -- now loop over all ewr units
             for n = 1, #active_ewr do
-              local ewr_controller = active_ewr[n]:getGroup():getController()
+              local ewr_controller = nil
+              if active_ewr[n]['is_awacs'] then
```
Comment (lukrop) [inline GCICAP.lua]: Let's use brackets and apostrophe only where they need to be used. Of course its valid syntax but I handled it that way in the whole project.

``` Lua
if active_ewr[n].is_awacs then
```

and

``` Lua
ewr_controller = active_ewr[n].unit:getController() 
```

File: GCICAP.lua [hunk 2]
```diff
@@ -976,7 +976,7 @@ do
               end
               -- and check if the EWR detected the aircraft
               if ewr_controller:isTargetDetected(ac, RADAR) then
-                ewr = active_ewr[n]
+                ewr = active_ewr[n]['unit']
```
Comment (lukrop) [inline GCICAP.lua]: Same here

``` Lua
ewr = active_ewr[n].unit
```

**Comment by lukrop** [comment]
Very nice fix! If you make those changes and test it one more time I'll merge it. Thank you!

**Comment by cjbehm** [comment]
Rebased and fixed the style comment, so I think we're set. Tested and still working.

**Pull request merged**